Makefile

```diff
@@ -11,12 +11,27 @@ test: compile
 compile:
 	@ test -d logs || mkdir logs
 	@ pdflatex --jobname=statutter -halt-on-error main.tex >> logs/compile \
-	  && echo "Compiled report" || (cat logs/compile && fail) 
+	  && echo "Compiled report" || (cat logs/compile && fail)
 
 open: compile
 	$(OPEN) statutter.pdf
 
 clean:
 	rm -f *.log *.aux *.lof *.pdf *.toc *.lot
+	rm -f gh-pages/index.html gh-pages/statutter.pdf
 
-.PHONY: compile open clean
+jekyll: clean gh-pages/index.html gh-pages/statutter.pdf
+
+publish: clean gh-pages/index.html gh-pages/statutter.pdf
+	rm -rf gh-pages/_site
+	ghp-import gh-pages
+	git push origin gh-pages -f
+
+gh-pages/index.html:
+	echo "---\nlayout: default\n---" > gh-pages/index.html
+	pandoc -f latex -t html content.tex >> gh-pages/index.html
```

Comment:
Do you want to move everything to a `content.tex` file or is this supposed to be `main.tex`?